src/scripts/move.js

```diff
@@ -0,0 +1,247 @@
+'use strict';
+
+// import
+const {
+  getNotEmptyFields,
+  getNotEmptyFieldsCoords,
+  checkBetween,
+  randomField,
+} = require('./utils');
+
+const gameScore = document.querySelector('.game-score');
+const gameField = document.querySelector('.game-field');
+const allCells = gameField.querySelectorAll('.field-cell');
+const highestScore = document.querySelector('.highest-score');
+const values = [2, 2, 2, 2, 2, 2, 2, 2, 2, 4];
```

Comment:
Do we really need this array? 🙂 As far as I understand it was used only for getting random start value, in this case can we just generate random number from 0 to 1 and to get same chance to get needed item we can return something like

```
randomValue <= 0.1
  ? 4
  : 2;
```

wdyt?